generative_ai/tune_code_generation_model.py

```diff
@@ -51,6 +58,8 @@ def tune_code_generation_model(
       train_steps: Number of training steps to use when tuning the model.
     """
     vertexai.init(project=project_id, location=location, credentials=credentials)
+    eval_spec = TuningEvaluationSpec(evaluation_data=evaluation_dataset)
+    eval_spec.tensorboard = aiplatform.Tensorboard(tensorboard_name=tensorboard_instance_name)

```

Comment:
```suggestion
    eval_spec = TuningEvaluationSpec(
        evaluation_data=evaluation_dataset,
        tensorboard=aiplatform.Tensorboard(
            tensorboard_name=tensorboard_instance_name
        ),
    )
```
Can this be used instead? Protos should be constructed with fields in the initialization , rather than constructing the individual fields separately.

Reply:
I prefer the original one. It clearly shows the Tensorboard logic.